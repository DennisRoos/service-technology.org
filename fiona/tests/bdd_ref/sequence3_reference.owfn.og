INTERFACE
  INPUT
;
  OUTPUT
    a,
    b,
    c;

NODES
  0 : (!a + !b + !c) : blue,
  1 : (!a + !b) : blue,
  2 : !a : blue,
  3 : final : blue : finalnode,
  4 : !b : blue,
  5 : (!a + !c) : blue,
  6 : !c : blue,
  7 : (!b + !c) : blue;

INITIALNODE
  0;

TRANSITIONS
  0 -> 1 : !c,
  0 -> 5 : !b,
  0 -> 7 : !a,
  1 -> 2 : !b,
  1 -> 4 : !a,
  2 -> 3 : !a,
  4 -> 3 : !b,
  5 -> 2 : !c,
  5 -> 6 : !a,
  6 -> 3 : !c,
  7 -> 4 : !c,
  7 -> 6 : !b;
