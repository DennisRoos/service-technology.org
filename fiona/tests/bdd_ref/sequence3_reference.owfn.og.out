digraph g1 {
graph [fontname="Helvetica", label="OG of ./sequence_suite/sequence3.owfn (parameters: -m1 -E)"];
node [fontname="Helvetica" fontsize=10];
edge [fontname="Helvetica" fontsize=10];
p0 [label="# 0\n(!a + !b + !c)", fontcolor=black, color=blue];
p0->p1 [label="!c", fontcolor=black, color=blue];
p1 [label="# 1\n(!a + !b)", fontcolor=black, color=blue];
p1->p2 [label="!b", fontcolor=black, color=blue];
p2 [label="# 2\n!a", fontcolor=black, color=blue];
p2->p3 [label="!a", fontcolor=black, color=blue];
p3 [label="# 3\nfinal", fontcolor=black, color=blue, peripheries=2];
p1->p4 [label="!a", fontcolor=black, color=blue];
p4 [label="# 4\n!b", fontcolor=black, color=blue];
p4->p3 [label="!b", fontcolor=black, color=blue];
p0->p5 [label="!b", fontcolor=black, color=blue];
p5 [label="# 5\n(!a + !c)", fontcolor=black, color=blue];
p5->p2 [label="!c", fontcolor=black, color=blue];
p5->p6 [label="!a", fontcolor=black, color=blue];
p6 [label="# 6\n!c", fontcolor=black, color=blue];
p6->p3 [label="!c", fontcolor=black, color=blue];
p0->p7 [label="!a", fontcolor=black, color=blue];
p7 [label="# 7\n(!b + !c)", fontcolor=black, color=blue];
p7->p4 [label="!c", fontcolor=black, color=blue];
p7->p6 [label="!b", fontcolor=black, color=blue];
}